samza-core/src/main/scala/org/apache/samza/container/TaskInstance.scala

```diff
@@ -203,6 +204,18 @@ class TaskInstance(
     trace("Checkpointing offsets for taskName: %s" format taskName)
 
     offsetManager.writeCheckpoint(taskName, checkpoint)
+
+    if (checkpoint != null) {
+      checkpoint.getOffsets.asScala.filter { case (ssp, _) =>
```

Comment:
Minor: formatting:

```
checkpoint.getOffsets.asScala
    .filter { ... }
    .grouBy { ... }
    .foreach { ... }
```

Reply:
Sure. I have updated the code as suggested.